src/javascripts/Views/addMastersModal.js

```diff
@@ -0,0 +1,31 @@
+ngapp.controller('addMastersModalController', function($scope, $timeout, errorService, modalService) {
+    let opts = $scope.modalOptions;
+    let selectedFile = opts.handle;
+    let selectedFileIndex = xelib.GetFileLoadOrder(selectedFile);
+    let allMasters =xelib.GetLoadOrder().split('\r\n');
```

Comment:
This isn't getting the filenames of the currently loaded files.  `GetLoadOrder` returns the user's `loadorder.txt`.  You want `xelib.GetLoadedFileNames`, which is defined in `xeFileValues.js`.  Note to self: it may make sense to move the function to `setup.js`.